flow-typed/types/playlist.js

```diff
@@ -12,9 +12,9 @@ declare type KPPlaylistOptions = _KPPlaylistOptions;
 
 /**
  * @typedef {Object} KPPlaylistCountdownOptions
- * @param {number} [timeToShow] - When the countdown will appear (by default is towards the end)
- * @param {number} [duration=10] - How match time the countdown will appear
- * @param {boolean} [showing=true] - Whether to show the countdown
+ * @property {number} [timeToShow] - Shows when the countdown is scheduled to appear (by default, this is towards the end).
+ * @property {number} [duration=10] - Shows how the match time countdown will appear.
```

Comment:
Yair not sure what you meant by this line - can you clarify?

Reply:
`duration` is a property which set how **much** time the countdown will displayed.  
is it clear now @AnnaAleksandrowicz ?

Reply:
Let's use "How long will the countdown appear" - OK?